vizro-core/src/vizro/models/_components/form/_text_area.py

```diff
@@ -0,0 +1,50 @@
+from typing import List, Literal
+
+import dash_bootstrap_components as dbc
+from dash import html
+
+try:
+    from pydantic.v1 import Field
+except ImportError:  # pragma: no cov
+    from pydantic import Field
+
+from vizro.models import Action, VizroBaseModel
+from vizro.models._action._actions_chain import _action_validator_factory
+from vizro.models._models_utils import _log_call
+
+
+class TextArea(VizroBaseModel):
+    """Component provided to `Form` to allow user multi-line text input.
+
+    Args:
+        type (Literal["text_area"]): Defaults to `"text_area"`.
+        title (str): Title to be displayed. Defaults to `""`.
+        placeholder (str): Default text to display in input field. Defaults to `""`.
+        actions (Optional[List[Action]]): Defaults to `[]`.
+    """
+
+    type: Literal["text_area"] = "text_area"
+    # TODO: before making public consider naming this field (or giving an alias) label instead of title
+    title: str = Field("", description="Title to be displayed")
+    placeholder: str = Field("", description="Default text to display in input field")
+    actions: List[Action] = []
+
+    # Re-used validators
+    _set_actions = _action_validator_factory("value")
```

Comment:
This way of configuring TextArea and UserInput: 
- `_set_actions = _action_validator_factory("value")`
- `debouncing=True`

causes that if the "actions" property is defined, actions chain will be triggered when the enter key is pressed or `when the component loses focus`.

If we want not to trigger actions chain every time when the component loses focus, but only if the enter key is pressed, then we should set `_action_validator_factory` argument to `n_submit`.